src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardSamUserDirectives.scala

```diff
@@ -29,7 +29,7 @@ trait StandardSamUserDirectives extends SamUserDirectives with LazyLogging with
     onSuccess {
       getActiveSamUser(oidcHeaders, userService, tosService, samRequestContext).unsafeToFuture()
     }.tmap { samUser =>
-      logger.info(s"Handling request for active Sam User: $samUser")
+      logger.debug(s"Handling request for active Sam User: $samUser")
```

Comment:
I made this change for the `asAdminServiceUser` directive I am adding in https://github.com/broadinstitute/sam/pull/1148 so it is consistent

Reply:
One of the things we lost with the switch to JWT was that we don't know which user is making any particular request anymore.  That's why this was added.  I admit that it's gross and not very graceful and is creating a lot of noise, but for troubleshooting, debugging, and production support, being able to get the `SamUser` info for a request is really valuable.  

Maybe the alternative is to just log out the `samRequestContext` (or at least part of it) when we log?  I have no idea what's in there or if that would be a terrible idea though.